docs/en-US/component/table.md

```diff
@@ -312,6 +312,7 @@ table/table-layout
 | current-change     | triggers when current row changes                                                                                                            | ^[Function]`(currentRow: any, oldCurrentRow: any) => void`                                   |
 | header-dragend     | triggers after changing a column's width by dragging the column header's border                                                              | ^[Function]`(newWidth: number, oldWidth: number, column: any, event: MouseEvent) => void`    |
 | expand-change      | triggers when user expands or collapses a row (for expandable table, second param is expandedRows; for tree Table, second param is expanded) | ^[Function]`(row: any, expandedRows: any[]) => void & (row: any, expanded: boolean) => void` |
+| scroll             | Invoked after scrolled                                                                                                                       | Object                                                                                       |
```

Comment:
The scroll type should be a function, and the parameter accepted by the function can be directly the object dispatched by the `scrollbar` component `{scrollTop: number, scrollLeft: number }`.

https://github.com/element-plus/element-plus/blob/84fb6933c39a51245911ea99245171477579ae73/packages/components/scrollbar/src/scrollbar.vue#L95-L98